Apps/FCM/ios/FCMSampleApp.xcodeproj/project.pbxproj

```diff
@@ -472,7 +472,7 @@
 				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 2YC97BQN3N;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = FCMSampleApp/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
```

Comment:
[All of our native iOS SDKs support 13.0](https://github.com/customerio/customerio-ios/blob/d43d24806fb2e14896c5efd4143bda177ba6c56b/CustomerIOMessagingPushFCM.podspec#L16). Why did this need to change?

Reply:
Our dependencies require a minimum deployment target of 13.0 but React native's default library `yoga` requires a minimum deployment target of `13.4` and hence same is expected in the PodSpec and updated in Xcode configuration. In fact, `@react-native-clipboard/clipboard` too requires 13.4.